### ISSUE TITLE
function documentation automatic completion issue

### ISSUE THREAD

**Issue by bbinet** [martinhoefling/molten#25]
When I start to type a function name in the "function" field in the "Execute" tab I get error message: 
```
Invalid function
no documentation found for <xxx>
```
See screenshot "invalid_function.png":
![invalid_function](https://cloud.githubusercontent.com/assets/76281/10423665/fe271acc-70c8-11e5-9ed7-3ce6d6ec97ef.png)

And if I empty the "function" field completely, I get the following screenshot "empty_function.png:
![empty_function](https://cloud.githubusercontent.com/assets/76281/10423683/272b52bc-70c9-11e5-8839-bfefad88ef7f.png)

**Comment by martinhoefling** [comment]
What salt version are you running on master and minions?

**Comment by bbinet** [comment]
```
$ salt --version
salt 2015.5.0 (Lithium)
```
(on both master and minions)

**Comment by martinhoefling** [comment]
Oh ok, so this won't work due to a salt bug. https://github.com/saltstack/salt/pull/25020. The 'invalid function' display is actually intended behavior since a function must always have a module and a dot as prefix. So I'm closing this.

**Comment by bbinet** [comment]
thanks @martinhoefling, I've temporarily monkey patched my salt master with saltstack/salt#25020 fix, and I confirm it fixes this function doc issue.
